src/DurableSDK/PowerShellExtensions.cs

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections;
+using System.Collections.ObjectModel;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
```

Comment:
Same as we did for `TypeExtensions`, I want to remove the dependencies of the `DurableSDK` to the worker. So here I copy over the PS-Extensions logic.